### PR TITLE
Conductivity and Chargeability 

### PR DESCRIPTION
**Pull request by sgkang**
Ready to be merged.

### CODE REVIEW

**Comment by lheagy** [comment]
It looks like there is an odd link (two http's) in one of the references that were added, so that is causing the fail 
build 791 - ubcgif em - travis ci 2016-02-16 19-38-21

**Comment by sgkang** [comment]
it seems because of eprint = {}. Getting closer to pass ha.

2016-02-16 19:39 GMT-08:00 Lindsey <notifications@github.com>:

**Comment by sgkang** [comment]
Hi, @dccowan and @dougoldenburg could you review?

File: examples/physical_properties/chargeability/ColeCole.ipynb [hunk 1]
```diff
@@ -0,0 +1,269 @@
+{
+ "cells": [
```
Comment (lheagy) [inline examples/physical_properties/chargeability/ColeCole.ipynb]: can we put this in the EM examples repository? https://github.com/ubcgif/em_examples

File: content/geophysical_surveys/index.rst [hunk 2]
```diff
@@ -15,23 +15,23 @@ Geophysical Surveys
 
 For each geophysical method, we will follow the framework:
 
-- Intro
```
Comment (lheagy) [inline content/geophysical_surveys/index.rst]: this is possibly the file where the merge conflict is happening. Do you want to try and resolve it? We can sit together if that would be helpful

Comment (sgkang) [inline content/geophysical_surveys/index.rst]: Sounds good. I'll drop by your office later in this afternoon.

2016-02-17 12:59 GMT-08:00 Lindsey <notifications@github.com>:

**Comment by lheagy** [comment]
the fail is because the alberta geologic survey changed their website, so the report we were referencing moved... so we will need to find it !

**Comment by sgkang** [comment]
I see. Thanks lindsey for tracking this about.

2016-03-04 10:30 GMT-08:00 Lindsey <notifications@github.com>:

File: examples/physical_properties/chargeability/ColeCole.ipynb (hunk 1 above)
Comment (sgkang) [inline examples/physical_properties/chargeability/ColeCole.ipynb]: Sound good I'll move this.

**Comment by sgkang** [comment]
@lheagy could you merge this to master?

**Comment by lheagy** [comment]
We are trying to get a round of review in before merging onto master (it is easier to make inline suggestions / comments on the file diff in the pull request than once they have been put onto master). I will put it on doug's branch though. We can take care of (hopefully all!?) the pull requests by next week

**Pull request closed without merging**